T012_M1_load_data.py

```diff
@@ -152,7 +152,6 @@ def student_school_dictionary(filepath: str) -> dict[list]:
     first_run = True
 
     for line in file:
-        temp_dict = {}
```

Comment:
Milan, I still need you to review the earliest pull request on that list (the one at the bottom) 👯‍♀️

Reply:
yep, thank you